tests/integration/product_create.t

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test;
+use Test::More;
+
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+
+remove_all_products();
+wait_dynamic_front();
+my $ua = new_client();
+create_user($ua, {});
+
+my %args_ref = (
+	code => '200000000099',
+	lang => "en",
+	product_name => "Test-75ml",
+	generic_name => "Tester",
+	quantity => "75 ml",
+	link => "https://github.com/openfoodfacts/openfoodfacts-server",
+	expiration_date => "test",
+	ingredients_text => "apple, milk",
+	origin => "france",
+	serving_size => "10g",
+	packaging_text => "no",
+	action => "process",
+	type => "add",
+	".submit" => "submit"
+);
+
+create_product($ua, \%args_ref);
```

Comment:
As discussed. We should make things more explicit, without loosing the concision in tests.

Your big hashmap could be in a TestDefaults.pm, name `$default_product_form_ref`

(I prefer the term `form` because it's what it is here: the values to fill a form)

```
my $form_ref = create_from_defaults($default_product_form_ref, {code => '200000000099'});
create_product($ua, $form_ref);
```

Reply:
Yup after I get this working on the CI once, will commit these changes